### DISCUSSION TITLE
Minik projeler için Repo önerisi.

### DISCUSSION THREAD

**Discussion by buraksenyurt** [thecoderverse/community#2] [Ideas]
Merhabalar,

Çok büyük çaplı olmayı bazen birkaç günde bitebilecek ya da antrenman amaçlı olan, tüm bunlara rağmen kod kalitesi yüksek uygulamaları toplayacağımız bir Repo açalım diyorum. Adı da **MiniVerse** olabilir :) Ne dersiniz?

**Comment by KardelRuveyda** [comment]
Günlük planlayıcı yapabiliriz :) ( https://structured.app/ ) buna benzer ama daha basit, kendi adıma benim çok ihtiyacım oluyor. Özellikle görevleri yaptıktan sonra üzerini çizdiğimde garip bir şekilde motive oluyorum :-D

**Comment by aynrkatircioglu** [comment]
Teams içine bot yazabiliriz. :) En çok neye ihtiyacınız oluyor gibi bir anket ile veri toplayıp daha sonra üzerine konuşabiliriz.

**Reply by buraksenyurt** [reply]
Bunu ve diğer fikirleri [MiniVerse](https://github.com/futureverseofficial/MiniVerse) reposundaki Readme'ye ekleyebiliriz.
